src/js/utils/MetronomeUtil.js

```diff
@@ -91,5 +91,36 @@ module.exports = {
     });
 
     return rootTree;
+  },
+
+  parseJob(job) {
+    let {history}= job;
+
+    if (history == null) {
+
```

Comment:
Defining a holistic set rules doesn't mean that we can't change them. It also doesn't mean that we should stop discussing these!  Defining them as eslint rules just ensures that we all, including third party developers, are aware of such rules. I'd suggest to start with a set of very well documented rules like those form [airbnb](https://github.com/airbnb/javascript), and adjust that set where wee see need. 

/cc @dcos/frontend